Yevhenii-Antonov/README.md

```diff
@@ -0,0 +1,43 @@
+GoogleDocs
+==========
+
+
+GoogleDocs is a web-based word processing tool provided by Google. It allows users to create, edit, and collaborate on documents online. This README.md provides an overview of GoogleDocs and its key features.
+
+## Features
+
+* Document Creation: Users can create new documents from scratch or choose from a variety of pre-designed templates.
```

Comment:
Consistency with time and verb forms used, either use present simple or present  continuous throughout, do not jump between times when not necessary.